### PR TITLE
Management of databases, without tests

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

File: src/lib/y2postgresql/databases_list.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      end
+    end
+
+    def read
+      list = Yast::Execute.on_target(
+        "sudo", "-u", "postgres", "/usr/bin/psql", "--list", stdout: :capture
+      )
+      return unless list
+
+      list.lines[3..-3].each do |line|
```
Comment (lslezak) [inline src/lib/y2postgresql/databases_list.rb]: The magic constants for skipping some lines does not look nice. Is there some `psql` parameters to skip them? Or another (machine readable) output format?

I have to admit than I'm not much familiar with postgresql...

**Comment by lslezak** [comment]
Otherwise it looks good.

File: src/lib/y2postgresql/databases_list.rb (hunk 1 above)
Comment (ancorgs) [reply src/lib/y2postgresql/databases_list.rb]: Yes, that intentionally... let's say... improvable :-)  I wanted to keep it supersimple, but I will add a disclaimer comment

Comment (lslezak) [reply src/lib/y2postgresql/databases_list.rb]: I see, with an appropriate comment it's OK :wink:

**Pull request closed without merging**